extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -406,39 +417,37 @@ protected function sendEmails( array $watchers, WallNotificationEntity $notifica
 				wfRunHooks( 'NotificationGetMailNotificationMessage', [
```

Comment:
Is there a way that `$data` above won't be populated after the call to this hook?